components/workspacekit/cmd/rings.go

```diff
@@ -486,9 +486,6 @@ var ring1Cmd = &cobra.Command{
 		slirpCmd.SysProcAttr = &syscall.SysProcAttr{
 			Pdeathsig: syscall.SIGKILL,
 		}
-		slirpCmd.Stdin = os.Stdin
```

Comment:
Do we not take input from os.Stdin for slirpCmd?

Reply:
Not that I know of. Slirp4net uses a socket to transport the data over the namespace boundary.